dataforest/config/MetaPlotMethods.py

```diff
@@ -9,4 +10,19 @@ def PLOT_METHOD_LOOKUP(cls):
 
     @property
     def PLOT_METHODS(cls):
```

Comment:
Good job picking up on this metaclass config pattern I was using. I was worried that it was confusing, but I'm glad you were able to figure it out.